create-release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Creates new package release in git.
+# a skip-merge is about not merge master to release.
+
+set -e
+set -o pipefail
+
+fail() {
+    echo "Error: $*" 1>&2
+    exit 1
+}
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ "$current_branch" != "release" ]]; then
+    echo 'Checkout "release".'
+    git checkout release
+fi
+
+if [[ "$1" == "skip-merge" ]]; then
+    echo 'Skip merge master to release'
+else
+    git merge master
+fi
+
+if ! git diff --quiet; then
+    fail 'Make sure you have no uncommitted changes in your repository.'
+fi
+
+echo "Current git commit is: $(git describe)"
+
+read -p 'Enter new version (e.g. "1.8.0"): ' version
+if [[ ! "$version" =~ ^[1-9][0-9]*\.[0-9]+\.[0-9]+$ ]]; then
+    fail 'Unexpected version format.'
+fi
+
+read -p 'Enter new release number: ' release
```

Comment:
You should always use `-r` option with read: https://github.com/koalaman/shellcheck/wiki/SC2162
It's not going to cause problems here though.

Reply:
Yes, use this script can modify spec and also create new srpm for bew build.

The "read -p" is copied from the pdc-server repo's codes it can make the echo and read in one line.
Like you said here we verify the input, the version just can be "x.y.z", release just can be a number. Both "read -p" and "echo / read -r" should work fine.  But I have changed to read -r to keep good use.